extensions/libraries/redcore/factory/factory.php

```diff
@@ -102,4 +102,51 @@ protected static function createDbo()
 
 		return $db;
 	}
+
+	/**
+	 * Create a mailer object
+	 *
+	 * @return  JMail object
+	 *
+	 * @see     JMail
+	 * @since   11.1
+	 */
+	protected static function createMailer()
```

Comment:
Maybe `getMailer` might be a better name? Also having it created only if instance doesn't exists already?